<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopInfoDlg</class>
 <widget class="QDialog" name="ShopInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#backgroundWidget
{
	border-image: url(:/shareImages/dlg_background.png);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="contentWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>329</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#contentWidget{
	background-color:#ffffff;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color: rgb(51, 51, 51);</string>
           </property>
           <property name="text">
            <string>组织名称：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="corpNameLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#ea8852;
font-family:&quot;微软雅黑&quot;;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color: rgb(51, 51, 51);</string>
           </property>
           <property name="text">
            <string>门店名称：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="shopNameLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#ea8852;
font-family:&quot;微软雅黑&quot;;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>18</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="minimumSize">
           <size>
            <width>152</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>197</width>
            <height>56</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font-family:&quot;微软雅黑&quot;;
	color:#ffffff;
	font-size:14px;
	border-image: url(:/shareImages/sure_normal.png);
}

QPushButton:pressed{
	border-image: url(:/shareImages/sure_press.png);
}
</string>
          </property>
          <property name="text">
           <string>确认上传</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancellButton">
          <property name="minimumSize">
           <size>
            <width>152</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>197</width>
            <height>56</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font-family:&quot;微软雅黑&quot;;
	color:#ffffff;
	font-size:14px;
	border-image: url(:/shareImages/cancel_normal.png);
}

QPushButton:pressed{
	border-image: url(:/shareImages/cancel_press.png);
}
</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
